### ISSUE TITLE
The admin-side Attendee list does not initially display any attendees

### ISSUE THREAD

**Issue by SkylerRingtail** [MidwestFurryFandom/rams#106]
The Attendees page (admin side; `People` -> `Attendees`) does not list all attendees at first; you have to click the page number button ("1") for it to display. Is this partly intentional to try and prevent excess database queries (i.e. let the user search first, if they want, rather than loading all the data at once)? If not, I would expect the list of attendees (or at least the first page of x results) to load when opening the activity.

![mff admin missing attendee list](https://cloud.githubusercontent.com/assets/28108495/25568410/de830d80-2dc7-11e7-90ad-6865a71f5718.png)

**Comment by kitsuta** [comment]
Yeah, this was definitely an optimization thing. In practice, wanting to do anything except search for a specific user is pretty rare. This bites us most often during an event, when we (MAGFest) would have over 40 laptops hitting that page at once.

**Comment by jmdawson** [comment]
This is acceptable for now. We can review this once we've seen how MFF feels about it in an operational system. Once we have more than a handful of test users, I suspect MAGFest's choices will make more sense.

**Comment by ianlayton** [comment]
Makes sense to leave this the way it is for optimization reason, especially as we begin to make the database bigger.
